extension.toml

```diff
@@ -14,4 +14,4 @@ language = "REDscript"
 
 [grammars.redscript]
 repository = "https://github.com/jac3km4/tree-sitter-redscript"
-commit = "08dd7c698c64b0d55265c4462ec3e32728a0d82e"
+rev = "master"
```

Comment:
should point at a commit rather than a branch (a branch might become incompatible with this commit)

Reply:
right, then this waits for merge on tree sitter